pyccel/codegen/printing/pycode.py

```diff
@@ -1136,6 +1136,13 @@ def _print_TypingFinal(self, expr):
         annotation = self._print(expr.arg)
         return f'const {annotation}'
 
+    def _print_ListPop(self, expr):
+        args = ""
+        if expr.pop_index:
+            args = self._print(expr.pop_index)
```

Comment:
Small suggestion. Instead of redefining the variable `args`, it is more Pythonic to use the single-liner
```suggestion
        args = self._print(expr.pop_index) if expr.pop_index else ""
```